gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -70,6 +89,68 @@ enum
   PROP_IS_UPDATABLE,
 };
 
+/**
+ * @brief to get and register hardware accelerator backend enum
+ */
+static GType
+accl_hw_get_type (void)
```

Comment:
recommended to move original code here,  and fix indent in line 105

Reply:
Moving the original code here does allow to use the G_GNUC_CONST attribute as this attribute can only be set in function declaration.

Reply:
Line 105 current indentation is the needed indentation to pass indent CI test.
```
static gchar *
strcpy2 ()
```
Updating to above fails CI indent test.